dalm/training/rag_e2e/train_rage2e.py

```diff
@@ -394,7 +394,7 @@ def train_e2e(
     # Potentially load in the weights and states from a previous save
     if resume_from_checkpoint:
         if resume_from_checkpoint is not None or resume_from_checkpoint != "":
-            accelerator.print(f"Resumed from checkpoint: {resume_from_checkpoint}")
+            logger.info(f"Resumed from checkpoint: {resume_from_checkpoint}")
```

Comment:
Any downsides of using `logger.info` instead of `accelerator.print`?  This logger was created using the `accelerate.logging.get_logger` function.  I'm assuming it will be the same, but just use the logging framework rather than print, and therefore be captured into cloudwatch logs